rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -46,19 +46,21 @@ object SystemProcesses {
   }
 
   def stdoutAck(
-      space: RhoISpace,
+      space: RhoISpace[Task],
       dispatcher: Dispatch[Task, ListParWithRandomAndPhlos, TaggedContinuation]
   ): Seq[ListParWithRandomAndPhlos] => Task[Unit] = {
     case Seq(ListParWithRandomAndPhlos(Seq(arg, ack), rand, _)) =>
       Task.now(Console.println(prettyPrinter.buildString(arg))).flatMap { (_: Unit) =>
```

Comment:
Switching to a for comprehension here might be a little more readable than the nested `flatMaps` here now